### DISCUSSION TITLE
How do you bulk import cables with multiple terminations?

### DISCUSSION THREAD

**Discussion by duplexer** [netbox-community/netbox#15493] [Q&A]
Hi there,

I'm trying to bulk import cables with multiple terminations but have been unsuccessful so far. I've attempted separating the individual terminations via commas and changing the CSV delimiter to a semicolon, and I've attempted using both JSON and YAML format to import and specify the multiple terminations that way. I haven't been able to figure out how to do this successfully though. Exporting a multi-termination cable that was created manually via the GUI indicates that it should be as simple as separating the terminations with a comma.

What's the best way to do this if it's even possible?

Thanks!

**Comment by duplexer** [comment]
Finally found a definitive answer that this is not possible via csv, json, or yaml.

**Reply by xkilian** [reply]
I have not done this yet, but I imagine the solution is to have a custom script or external script(via api) be responsible to create the cables.
